packages/library-legacy/src/connection.ts

```diff
@@ -58,6 +58,19 @@ import type {Blockhash} from './blockhash';
 import type {FeeCalculator} from './fee-calculator';
 import type {TransactionSignature} from './transaction';
 import type {CompiledInstruction} from './message';
+/** EXPERIMENTAL: Imports */
+import {Base58EncodedAddress} from '../../keys';
+import {SolanaRpcMethods} from '../../rpc-core/src';
+import {Blockhash as EXPERIMENTAL_Blockhash} from '../../rpc-core/src/blockhash';
+import {TransactionSignature as EXPERIMENTAL_TransactionSignature} from '../../rpc-core/src/transaction-signature';
+import {
+  Commitment as EXPERIMENTAL_Commitment,
+  Slot,
+  U64UnsafeBeyond2Pow53Minus1,
+} from '../../rpc-core/src/rpc-methods/common';
+import {createDefaultRpcTransport, createSolanaRpc} from '../../library/src';
```

Comment:
Try to do this entire PR without ever importing `library/`. The point is for `library/` to have all of the modern configuration (like the `bigint` coalescer) and for `library-legacy/` to assemble all of the primitives in a ‘legacy compatibility’ style.

Reply:
that makes a lot of sense for the coalescing no doubt